css/structure/jquery.mobile.forms.select.css

```diff
@@ -66,6 +66,10 @@
 .ui-select .ui-btn-icon-right .ui-li-count {
 	right: 3.2em;
 }
+/*Fixes #7918 : Count indicator position must be different for mini version*/
+.ui-select.ui-mini .ui-btn-icon-right .ui-li-count {
+  right: 2.6em;
```

Comment:
Indent using tabs, not spaces:
http://contribute.jquery.org/style-guide/css/#spacing